apps/src/gamelab/GameLabGroup.js

```diff
@@ -111,8 +111,10 @@ module.exports.Group = function (baseConstructor) {
   array.callMethodEach = function (methodName) {
     // Copy all arguments after the first parameter into methodArgs:
     var methodArgs = Array.prototype.slice.call(arguments, 1);
-    for (var i = 0; i < this.length; i++) {
-      this[i][methodName].apply(this[i], methodArgs);
+    // Use a copy of the array in case the method modifies the group
```

Comment:
Curious, could this also have used the simple trick in which you iterate backwards instead of forwards through the array?

Reply:
That would have fixed the bug for `destroy`, but I was trying to guard against any future method that removes a different sprite/more than one sprite from the group.